[13.0][DEL] Remove coverage

StefanRijnhart opened the pull request:
Upload currently fails with
```
Could not submit coverage: 422 Client Error:
Unprocessable Entity for url: https://coveralls.io/api/v1/jobs
```

Even though coverage data is gathered successfully in the correct format.
(See https://github.com/OCA/OpenUpgrade/runs/1738041013#step:13:18
for a run with coveralls debug output)

Having coverage on the migration scripts hardly makes any sense, as it only
displays which code does not run during the migration.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

pedrobaeza commented on .github/workflows/test.yml:
```diff
@@ -70,8 +69,4 @@ jobs:
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: coveralls
+          OPENUPGRADE_TESTS=1 $ODOO --database=$DB --update=$MODULES_NEW --db_host=$DB_HOST --db_port=$DB_PORT --db_port=$DB_PORT --db_password=$DB_PASSWORD --db_user=$DB_USERNAME --stop-after-init
```
There's two times db port, although it was the before, but good moment to amend it.

StefanRijnhart committed:
[DEL] Remove coverage

Upload currently fails with Could not submit coverage:
422 Client Error:
Unprocessable Entity for url: https://coveralls.io/api/v1/jobs

Even though coverage data is gathered successfully in the correct format.

Having coverage on the migration scripts hardly makes any sense, as it only
displays which code does not run during the migration.

pedrobaeza reviewed (approved):
Is this still draft?

StefanRijnhart commented:
Green, so it's ready.

pedrobaeza commented:
/ocabot merge nobump

pedrobaeza commented:
Merging manually
